myapp/handlers/auth-handlers.go

```diff
@@ -54,12 +54,16 @@ func (h *Handlers) PostUserLogin(w http.ResponseWriter, r *http.Request) {
 
 	_, err = h.Models.SpotifyTokens.GetSpotifyTokenForUser(user.ID)
 	if err != nil {
-		fmt.Println("Error getting Spotify token for user", err)
+		fmt.Println("Spotify token for user does not exist. Going to /users/spotauth to get one.", err)
 		http.Redirect(w, r, "/users/spotauth", http.StatusSeeOther)
 		return
 	}
 
 	h.SetSpotifyArtistsForUser(user.ID)
+	if err != nil {
```

Comment:
This err isn't updated from previous assignment on line 55
At this point, it will follow the same pattern as the err check on 56